modules/sdk-coin-vet/test/transactionBuilder/transferBuilder.ts

```diff
@@ -138,6 +138,34 @@ describe('Vet Transfer Transaction', () => {
         should.equal(toJson.gasPriceCoef, 128);
         should.equal(toJson.expiration, 64);
       });
+
+      it('should build a unsigned tx then add sender sig and build again', async function () {
+        const transaction = new Transaction(coins.get('tvet'));
+        const txBuilder = factory.getTransferBuilder(transaction);
+        txBuilder.sender(testData.addresses.validAddresses[2]);
+        txBuilder.recipients(testData.recipients);
+        txBuilder.gas(21000);
+        txBuilder.nonce(64248);
+        txBuilder.blockRef('0x014ead140e77bbc1');
+        txBuilder.addFeePayerAddress(testData.feePayer.address);
+        txBuilder.expiration(64);
+        txBuilder.gasPriceCoef(128);
+        const tx = (await txBuilder.build()) as Transaction;
+        const unsignedSerializedTx = tx.toBroadcastFormat();
+
+        const builder1 = factory.from(unsignedSerializedTx);
+        builder1.addSenderSignature(Buffer.from(testData.senderSig, 'hex'));
+        const senderSignedTx = await builder1.build();
+        const senderSignedSerializedTx = senderSignedTx.toBroadcastFormat();
+        should.equal(senderSignedSerializedTx, testData.senderSignedSerializedTxHex);
+
+        const builder2 = factory.from(testData.senderSignedSerializedTxHex);
+        builder2.addSenderSignature(Buffer.from(testData.senderSig, 'hex'));
```

Comment:
why do you need add this again? As sender signature was already added, so now, only feePayerSignature should be added, right?